src/Sylius/Bundle/ApiBundle/Filter/TaxonFilter.php

```diff
@@ -51,15 +51,10 @@ public function filterProperty(
         $queryBuilder
             ->distinct()
             ->addSelect('productTaxon')
-            ->join(sprintf('%s.productTaxons', $alias), 'p')
-            ->innerJoin($alias . '.productTaxons', 'productTaxon')
-            ->innerJoin('p.taxon', 'taxon')
-            ->andWhere('taxon.left >= :taxonLeft')
-            ->andWhere('taxon.right <= :taxonRight')
```

Comment:
I temporarily removed these wheres as they cause a problem mentioned by @arti0090 and if I see correctly all tests work after this change so it could be safe enough to go, but I'm not sure 🤔 wdyt @lchrusciel @Zales0123 ?

Reply:
If all our tests work, I'm ok wit that. However, it was added by @AdamKasp. Do you remember why did you add these lines in #12534?